docs/api/sql/Function.md

```diff
@@ -2530,7 +2530,7 @@ PROJCS["WGS 84 / Pseudo-Mercator",
 Format: 
 
 ```
-ST_Transform (A: Geometry, SourceCRS: String, TargetCRS: String, [Optional] lenientMode: Boolean)
+ST_Transform (A: Geometry, [Optional] SourceCRS: String, TargetCRS: String, [Optional] lenientMode: Boolean)
```

Comment:
Please provide overloaded variants of this function. Don't just specify `optional`.